src/Generating/FinishGen.h

```diff
@@ -70,6 +70,71 @@ class cFinishGenNetherClumpFoliage :
 
 
 
+class cFinishGenClumpTopBlock :
+	public cFinishGen
+{
+public:
+	// Contains the meta, type and weight for a clump block
+	struct BlockInfo
```

Comment:
> or whatever else

Whether or not xoft has a blocked drain.


On 7 Mar 2016, at 10:40, "Mattes D" <notifications@github.com> wrote:

> In src/Generating/FinishGen.h:
> 
> > @@ -70,6 +70,71 @@ class cFinishGenNetherClumpFoliage :
> >  
> >  
> >  
> > +class cFinishGenClumpTopBlock :
> > +	public cFinishGen
> > +{
> > +public:
> > +	// Contains the meta, type and weight for a clump block
> > +	struct BlockInfo
> Still, Foliage is the primary function, so it might be worth using it as the name.
> 
> The point is, when you later reference this as block, it creates a lot of confusion because it's unclear whether a block is this foliage info, or a topmost block in the chunk, or whatever else.
> 
> —
> Reply to this email directly or view it on GitHub.
>